docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md

```diff
@@ -2,6 +2,7 @@
 
 | Version | Date |
 |---|---|
+| [v.2.3.0](Data_Dictionary_Release_Notes.md#v220) | December 21, 2020 |
```

Comment:
Change the anchor to "#v230" and the date to January 2021 (since we don't know exactly when the release will happen).

Reply:
Okay updated, committed and pushed